app/bettermail.go

```diff
@@ -274,6 +285,9 @@ func handleCommitCommentPayload(payload CommitCommentPayload, c context.Context)
 		SenderUserName: senderUserName,
 		Subject: subject,
 		HTMLBody: mailHtml.String(),
+		Headers: map [string]string {
+			"In-Reply-To": messageId,
```

Comment:
Should this check to see if `messageId` is non-nil/empty, to handle datastore entries that were created before this change?